src/components/UserAddPopup.jsx

```
@@ -44,15 +48,17 @@ const UserAddPopup = ({
         if (!value) return "Last name is required";
         break;
       case "email":
-        if (!value) return "Email address is required";
+        const emailPattern = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)